### PR TITLE
Add documentation for `implements` in the class chapter

### PR DESCRIPTION
**Pull request by zhengbli**

### CODE REVIEW

File: pages/Classes.md [hunk 1]
```diff
@@ -390,3 +390,45 @@ interface Point3d extends Point {
...
+    name: string;
+    age: number;
+    learn: () => void;
+}
+
+interface Programmer {
+    favoriteLanguage: string;
+}
+
+class CoolKid implements Kid, Programmer {
```
Comment (DanielRosenwasser) [inline pages/Classes.md]: :+1: for the example

File: pages/Classes.md [hunk 2]
```diff
@@ -390,3 +390,45 @@ interface Point3d extends Point {
 var point3d: Point3d = {x: 1, y: 2, z: 3};
 ```
 
+## Implementing interfaces
+A class declaration can also implement interfaces in the `implements` clause.
```
Comment (DanielRosenwasser) [inline pages/Classes.md]: Newline above

File: pages/Classes.md [hunk 3]
```diff
@@ -390,3 +390,45 @@ interface Point3d extends Point {
...
+The class needs to provide an implementation for the properties of every interface in the `implements` clause.
```
Comment (DanielRosenwasser) [inline pages/Classes.md]: of every interface that it `implements`

File: pages/Classes.md [hunk 4]
```diff
@@ -390,3 +390,45 @@ interface Point3d extends Point {
...
+
+```ts
+interface Kid {
+    name: string;
+    age: number;
+    learn: () => void;
```
Comment (DanielRosenwasser) [inline pages/Classes.md]: learn(): void;

**Comment by DanielRosenwasser** [comment]
I realized that there already is documentation for this in the interfaces chapter. :/

**Comment by zhengbli** [comment]
Oops, I'll take a look to see if I can merge it somehow.

**Pull request closed without merging**